package.json

```diff
@@ -60,6 +60,7 @@
         "eslint-plugin-tsdoc": "^0.2.5",
         "highlight.js": ">=10.4.1",
         "husky": "^4.2.5",
+        "marked": "^4.0.12",
```

Comment:
what is this for?

Reply:
seems related to markdown https://www.npmjs.com/package/marked

Reply:
https://github.com/alexa/ask-sdk-controls/security/dependabot/4

Yes it was to remove this vulnerability